packages/trader/package-lock.json

```diff
@@ -978,6 +978,11 @@
         "md5": "^2.2.1"
       }
     },
+    "@contentpass/zxcvbn": {
```

Comment:
why trader `package-lock.json` is being updated?

Reply:
i was installing p2p to trader, and uninstall it, i think it might be some artifacts from that

Reply:
ah I see.. can you please make sure it's not being added in trader? I believe all of these belongs to `component` package 🤔

Reply:
@kelly-binary @mustofa-binary Won't P2P be in Cashier? If yes, then it would be in Core if I'm not mistaken

Reply:
@mahdipourziaei-binary @mustofa-binary oo yeah, p2p should be added to core. I was saying the packages (`storybook`, `file-dropzone`, `react-dropzone`, `file-selector`, `zxcvbn`) here looks like they belonged to components

Reply:
yes, i was just want to test it back then, i have removed p2p in trader, however, even i run npm run bootstrap, the additions log lines are still there, is it bcause previously we dont install components inside trader?